fifo_cache.py

```
@@ -0,0 +1,28 @@
+    def __init__(self, capacity: int):
+        self.capacity = capacity
+        self.cache = dict()  # 问题1：应为 OrderedDict，dict无序
+
+    def get(self, key: any) -> any:
+        return self.cache.get(ket)  # 问题2：变量名拼写错误，正确应该是key
+
+    def put(self, key: any, value: any) -> None:
+        if key in self.cache:
+            self.cache[key] = value
+        else:
+            # 问题3：边界判断写反，应该是 >
```

Ruff (0.14.6)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)